Test failures with trio 0.27.0

Hi!

After updating trio to 0.27.0 we are seeing the following test failure on trio-asyncio.

```
trio-asyncio> ============================= test session starts ==============================
trio-asyncio> platform linux -- Python 3.12.8, pytest-8.3.3, pluggy-1.5.0
trio-asyncio> rootdir: /build/source
trio-asyncio> configfile: pyproject.toml
trio-asyncio> plugins: trio-0.8.0
trio-asyncio> collected 123 items                                                            
trio-asyncio> 
trio-asyncio> tests/aiotest/test_add_reader.py ...                                     [  2%]
trio-asyncio> tests/aiotest/test_callback.py ...                                       [  4%]
trio-asyncio> tests/aiotest/test_coroutine.py ..                                       [  6%]
trio-asyncio> tests/aiotest/test_network.py .                                          [  7%]
trio-asyncio> tests/aiotest/test_thread.py ....                                        [ 10%]
trio-asyncio> tests/aiotest/test_timer.py ..                                           [ 12%]
trio-asyncio> tests/interop/test_adapter.py .................                          [ 26%]
trio-asyncio> tests/interop/test_calls.py ....................                         [ 42%]
trio-asyncio> tests/test_aio_subprocess.py ..............                              [ 53%]
trio-asyncio> tests/test_concurrent.py ...                                             [ 56%]
trio-asyncio> tests/test_deprecate.py ...........                                      [ 65%]
trio-asyncio> tests/test_misc.py ...............F.                                     [ 78%]
trio-asyncio> tests/test_sync.py .                                                     [ 79%]
trio-asyncio> tests/test_trio_asyncio.py .........FF..............                     [100%]
trio-asyncio> 
trio-asyncio> =================================== FAILURES ===================================
trio-asyncio> __________________________ test_run_trio_task_errors ___________________________
trio-asyncio>   + Exception Group Traceback (most recent call last):
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 341, in from_call
trio-asyncio>   |     result: TResult | None = func()
trio-asyncio>   |                              ^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 242, in <lambda>
trio-asyncio>   |     lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise
trio-asyncio>   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 182, in _multicall
trio-asyncio>   |     return outcome.get_result()
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_result.py", line 100, in get_result
trio-asyncio>   |     raise exc.with_traceback(exc.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 92, in pytest_runtest_call
trio-asyncio>   |     yield from thread_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 68, in thread_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 95, in pytest_runtest_call
trio-asyncio>   |     yield from unraisable_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 70, in unraisable_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 846, in pytest_runtest_call
trio-asyncio>   |     yield from self._runtest_for(item, "call")
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 829, in _runtest_for
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/capture.py", line 880, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/skipping.py", line 257, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 174, in pytest_runtest_call
trio-asyncio>   |     item.runtest()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 1627, in runtest
trio-asyncio>   |     self.ihook.pytest_pyfunc_call(pyfuncitem=self)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
trio-asyncio>   |     raise exception.with_traceback(exception.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 159, in pytest_pyfunc_call
trio-asyncio>   |     result = testfunction(**testargs)
trio-asyncio>   |              ^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 348, in wrapper
trio-asyncio>   |     return run(partial(fn, **kwargs), clock=clock, instruments=instruments)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 2407, in run
trio-asyncio>   |     raise runner.main_task_outcome.error
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 426, in _bootstrap_fixtures_and_run_test
trio-asyncio>   |     raise test_ctx.error_list[0]
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 193, in _fixture_manager
trio-asyncio>   |     async with trio.open_nursery() as nursery_fixture:
trio-asyncio>   |                ^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 1039, in __aexit__
trio-asyncio>   |     raise combined_error_from_nursery
trio-asyncio>   | ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio>   +-+---------------- 1 ----------------
trio-asyncio>     | Traceback (most recent call last):
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 195, in _fixture_manager
trio-asyncio>     |     yield nursery_fixture
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 250, in run
trio-asyncio>     |     await self._func(**resolved_kwargs)
trio-asyncio>     |   File "/build/source/tests/test_misc.py", line 336, in test_run_trio_task_errors
trio-asyncio>     |     with trio.testing.RaisesGroup(SystemExit, strict=False):
trio-asyncio>     |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>     | TypeError: RaisesGroup.__init__() got an unexpected keyword argument 'strict'
trio-asyncio>     +------------------------------------
trio-asyncio> ___________________ test_cancel_loop_with_tasks[True-False] ____________________
trio-asyncio>   + Exception Group Traceback (most recent call last):
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 341, in from_call
trio-asyncio>   |     result: TResult | None = func()
trio-asyncio>   |                              ^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 242, in <lambda>
trio-asyncio>   |     lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise
trio-asyncio>   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 182, in _multicall
trio-asyncio>   |     return outcome.get_result()
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_result.py", line 100, in get_result
trio-asyncio>   |     raise exc.with_traceback(exc.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 92, in pytest_runtest_call
trio-asyncio>   |     yield from thread_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 68, in thread_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 95, in pytest_runtest_call
trio-asyncio>   |     yield from unraisable_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 70, in unraisable_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 846, in pytest_runtest_call
trio-asyncio>   |     yield from self._runtest_for(item, "call")
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 829, in _runtest_for
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/capture.py", line 880, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/skipping.py", line 257, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 174, in pytest_runtest_call
trio-asyncio>   |     item.runtest()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 1627, in runtest
trio-asyncio>   |     self.ihook.pytest_pyfunc_call(pyfuncitem=self)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
trio-asyncio>   |     raise exception.with_traceback(exception.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 159, in pytest_pyfunc_call
trio-asyncio>   |     result = testfunction(**testargs)
trio-asyncio>   |              ^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 348, in wrapper
trio-asyncio>   |     return run(partial(fn, **kwargs), clock=clock, instruments=instruments)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 2407, in run
trio-asyncio>   |     raise runner.main_task_outcome.error
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 426, in _bootstrap_fixtures_and_run_test
trio-asyncio>   |     raise test_ctx.error_list[0]
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 193, in _fixture_manager
trio-asyncio>   |     async with trio.open_nursery() as nursery_fixture:
trio-asyncio>   |                ^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 1039, in __aexit__
trio-asyncio>   |     raise combined_error_from_nursery
trio-asyncio>   | ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio>   +-+---------------- 1 ----------------
trio-asyncio>     | Traceback (most recent call last):
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 195, in _fixture_manager
trio-asyncio>     |     yield nursery_fixture
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 250, in run
trio-asyncio>     |     await self._func(**resolved_kwargs)
trio-asyncio>     |   File "/build/source/tests/test_trio_asyncio.py", line 126, in test_cancel_loop_with_tasks
trio-asyncio>     |     catcher = trio.testing.RaisesGroup(
trio-asyncio>     |               ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>     | TypeError: RaisesGroup.__init__() got an unexpected keyword argument 'strict'
trio-asyncio>     +------------------------------------
trio-asyncio> ____________________ test_cancel_loop_with_tasks[True-True] ____________________
trio-asyncio>   + Exception Group Traceback (most recent call last):
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 341, in from_call
trio-asyncio>   |     result: TResult | None = func()
trio-asyncio>   |                              ^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 242, in <lambda>
trio-asyncio>   |     lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise
trio-asyncio>   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 182, in _multicall
trio-asyncio>   |     return outcome.get_result()
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_result.py", line 100, in get_result
trio-asyncio>   |     raise exc.with_traceback(exc.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 92, in pytest_runtest_call
trio-asyncio>   |     yield from thread_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/threadexception.py", line 68, in thread_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 95, in pytest_runtest_call
trio-asyncio>   |     yield from unraisable_exception_runtest_hook()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/unraisableexception.py", line 70, in unraisable_exception_runtest_hook
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 846, in pytest_runtest_call
trio-asyncio>   |     yield from self._runtest_for(item, "call")
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/logging.py", line 829, in _runtest_for
trio-asyncio>   |     yield
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/capture.py", line 880, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
trio-asyncio>   |     teardown.throw(outcome._exception)
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/skipping.py", line 257, in pytest_runtest_call
trio-asyncio>   |     return (yield)
trio-asyncio>   |             ^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/runner.py", line 174, in pytest_runtest_call
trio-asyncio>   |     item.runtest()
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 1627, in runtest
trio-asyncio>   |     self.ihook.pytest_pyfunc_call(pyfuncitem=self)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_hooks.py", line 513, in __call__
trio-asyncio>   |     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
trio-asyncio>   |     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
trio-asyncio>   |     raise exception.with_traceback(exception.__traceback__)
trio-asyncio>   |   File "/nix/store/yw83gmzrnj3ymclk3160qxn9rbra42hs-python3.12-pluggy-1.5.0/lib/python3.12/site-packages/pluggy/_callers.py", line 103, in _multicall
trio-asyncio>   |     res = hook_impl.function(*args)
trio-asyncio>   |           ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/bzg3wwihj3njvmyzjd3735ja7fq43aid-python3.12-pytest-8.3.3/lib/python3.12/site-packages/_pytest/python.py", line 159, in pytest_pyfunc_call
trio-asyncio>   |     result = testfunction(**testargs)
trio-asyncio>   |              ^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 348, in wrapper
trio-asyncio>   |     return run(partial(fn, **kwargs), clock=clock, instruments=instruments)
trio-asyncio>   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 2407, in run
trio-asyncio>   |     raise runner.main_task_outcome.error
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 426, in _bootstrap_fixtures_and_run_test
trio-asyncio>   |     raise test_ctx.error_list[0]
trio-asyncio>   |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 193, in _fixture_manager
trio-asyncio>   |     async with trio.open_nursery() as nursery_fixture:
trio-asyncio>   |                ^^^^^^^^^^^^^^^^^^^
trio-asyncio>   |   File "/nix/store/vyq13wihh1spxzk8yhd7y5rhqv4ybfbp-python3.12-trio-0.27.0/lib/python3.12/site-packages/trio/_core/_run.py", line 1039, in __aexit__
trio-asyncio>   |     raise combined_error_from_nursery
trio-asyncio>   | ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio>   +-+---------------- 1 ----------------
trio-asyncio>     | Traceback (most recent call last):
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 195, in _fixture_manager
trio-asyncio>     |     yield nursery_fixture
trio-asyncio>     |   File "/nix/store/d33kgdzsi31gqwv85g3xilk4b4vliafm-python3.12-pytest-trio-0.8.0/lib/python3.12/site-packages/pytest_trio/plugin.py", line 250, in run
trio-asyncio>     |     await self._func(**resolved_kwargs)
trio-asyncio>     |   File "/build/source/tests/test_trio_asyncio.py", line 126, in test_cancel_loop_with_tasks
trio-asyncio>     |     catcher = trio.testing.RaisesGroup(
trio-asyncio>     |               ^^^^^^^^^^^^^^^^^^^^^^^^^
trio-asyncio>     | TypeError: RaisesGroup.__init__() got an unexpected keyword argument 'strict'
trio-asyncio>     +------------------------------------
trio-asyncio> =========================== short test summary info ============================
trio-asyncio> FAILED tests/test_misc.py::test_run_trio_task_errors - ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio> FAILED tests/test_trio_asyncio.py::test_cancel_loop_with_tasks[True-False] - ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio> FAILED tests/test_trio_asyncio.py::test_cancel_loop_with_tasks[True-True] - ExceptionGroup: Exceptions from Trio nursery (1 sub-exception)
trio-asyncio> ======================== 3 failed, 120 passed in 4.11s =========================
```

--- a/.readthedocs.yml
+++ b/.readthedocs.yml
@@ -15,4 +15,5 @@ python:
     - requirements: docs-requirements.txt
 
 sphinx:
+  configuration: docs/source/conf.py
   fail_on_warning: true

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -132,7 +132,6 @@ todo_include_todos = False
 # html_theme_options.
 import sphinx_rtd_theme
 html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the

--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -3,4 +3,4 @@ pytest-cov
 pytest-trio
 outcome
 pytest-timeout
-trio >= 0.25.0
+trio >= 0.26.0

--- a/tests/test_misc.py
+++ b/tests/test_misc.py
@@ -333,9 +333,9 @@ async def test_run_trio_task_errors(monkeypatch):
     )
     expected = [ValueError("hi"), ValueError("lo"), KeyError(), IndexError()]
     await raise_in_aio_loop(expected[0])
-    with trio.testing.RaisesGroup(SystemExit, strict=False):
+    with trio.testing.RaisesGroup(SystemExit, flatten_subgroups=True):
         await raise_in_aio_loop(SystemExit(0))
-    with trio.testing.RaisesGroup(SystemExit, strict=False) as result:
+    with trio.testing.RaisesGroup(SystemExit, flatten_subgroups=True) as result:
         await raise_in_aio_loop(BaseExceptionGroup("", [expected[1], SystemExit()]))
 
     assert len(result.value.exceptions) == 1

--- a/tests/test_trio_asyncio.py
+++ b/tests/test_trio_asyncio.py
@@ -124,7 +124,7 @@ async def test_cancel_loop_with_tasks(autojump_clock, shield, body_raises):
 
     if body_raises:
         catcher = trio.testing.RaisesGroup(
-            trio.testing.Matcher(ValueError, match="hi"), strict=False
+            trio.testing.Matcher(ValueError, match="hi"), flatten_subgroups=True
         )
     else:
         catcher = contextlib.nullcontext()

